installer/test/CreateInstallerTestPackages/CreateInstallerTestPackages.vcxproj

```diff
@@ -0,0 +1,165 @@
+    <NMakeOutput>$(OutputPath)Microsoft.ProjectReunion.Test.InstallerPackages</NMakeOutput>
+    <NMakeCleanCommandLine>nmake /C /f makefile.mak ProjectDir=$(ProjectDir) SolutionDir=$(SolutionDir) OutDir=$(OutDir) TargetName=$(TargetName) clean</NMakeCleanCommandLine>
+    <NMakeReBuildCommandLine>nmake /C /f makefile.mak ProjectDir=$(ProjectDir) SolutionDir=$(SolutionDir) OutDir=$(OutDir) TargetName=$(TargetName)  rebuild</NMakeReBuildCommandLine>
+    <NMakePreprocessorDefinitions>WIN32;NDEBUG;$(NMakePreprocessorDefinitions)</NMakePreprocessorDefinitions>
+    <OutDir>$(OutDir)</OutDir>
+    <IntDir>$(IntDir)</IntDir>
+    <NMakeIncludeSearchPath>$(SolutionDir)tools</NMakeIncludeSearchPath>
+  </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Release|ARM'">
+    <NMakeBuildCommandLine>nmake /f makefile.mak ProjectDir=$(ProjectDir) SolutionDir=$(SolutionDir) OutDir=$(OutDir) TargetName=$(TargetName) </NMakeBuildCommandLine>
+    <NMakeOutput>$(OutputPath)Microsoft.ProjectReunion.Test.InstallerPackages</NMakeOutput>
+    <NMakeCleanCommandLine>nmake /f makefile.mak ProjectDir=$(ProjectDir) SolutionDir=$(SolutionDir) OutDir=$(OutDir) TargetName=$(TargetName) clean</NMakeCleanCommandLine>
+    <NMakeReBuildCommandLine>nmake /f makefile.mak ProjectDir=$(ProjectDir) SolutionDir=$(SolutionDir) OutDir=$(OutDir) TargetName=$(TargetName)  rebuild</NMakeReBuildCommandLine>
+    <NMakePreprocessorDefinitions>WIN32;NDEBUG;$(NMakePreprocessorDefinitions)</NMakePreprocessorDefinitions>
+    <OutDir>$(OutDir)</OutDir>
+    <IntDir>$(IntDir)</IntDir>
+    <NMakeIncludeSearchPath>$(SolutionDir)tools</NMakeIncludeSearchPath>
+  </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Release|ARM64'">
+    <NMakeBuildCommandLine>nmake /f makefile.mak ProjectDir=$(ProjectDir) SolutionDir=$(SolutionDir) OutDir=$(OutDir) TargetName=$(TargetName) </NMakeBuildCommandLine>
+    <NMakeOutput>$(OutputPath)Microsoft.ProjectReunion.Test.InstallerPackages</NMakeOutput>
+    <NMakeCleanCommandLine>nmake /f makefile.mak ProjectDir=$(ProjectDir) SolutionDir=$(SolutionDir) OutDir=$(OutDir) TargetName=$(TargetName) clean</NMakeCleanCommandLine>
+    <NMakeReBuildCommandLine>nmake /f makefile.mak ProjectDir=$(ProjectDir) SolutionDir=$(SolutionDir) OutDir=$(OutDir) TargetName=$(TargetName)  rebuild</NMakeReBuildCommandLine>
+    <NMakePreprocessorDefinitions>WIN32;NDEBUG;$(NMakePreprocessorDefinitions)</NMakePreprocessorDefinitions>
+    <OutDir>$(OutDir)</OutDir>
+    <IntDir>$(IntDir)</IntDir>
+    <NMakeIncludeSearchPath>$(SolutionDir)tools</NMakeIncludeSearchPath>
+  </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Release|x64'">
+    <NMakeBuildCommandLine>nmake /C /f makefile.mak ProjectDir=$(ProjectDir) SolutionDir=$(SolutionDir) OutDir=$(OutDir) TargetName=$(TargetName) </NMakeBuildCommandLine>
```

Comment:
nmake? msbuild is likely able to do anything you needed to do with nmake, but the syntax is clumsier. Revisit this at some point - we should reduce the number of unique build mechanisms being used (cc @EHO-makai )

Reply:
> Can we avoid committing in the .msix files? Git doesn't like binaries.

The broader issue is now referenced in #521, but here's how I'm addressing it here:

I originally was using the MSTest cert that was checked into /build to build these on-the-fly (which was expired and was using Howard's updated copy locally, but as that is not checked in and may not, F5 for the installer would be broken if I didnt include the actually valid signed packages. I wanted to have a reliable build of the installer with tests that anyone could include and simply F5, including those without access to the MSTest cert to verify local builds of the installer. These packages also don't change unless adding more tests and have no executable code, so including them does speed up build times, so I changed it to be a separate solution to build the test packages that can then be added as test collateral when needed.

I could update this to get the cert from Azure every time and build the packages with every installer build, but that would add significantly to build time and break F5 by requiring the developer to run a pipeline. And it coudl also cause the solution to not work for folks without access to the MSTest cert. The current solution should work for everyone to build and validate the installer with a simple F5 and then have it include their own packages with an override header without need of pipeline access or execution. I've added #521 discuss this problem. When there's a good solution here I'll be happy to make the changes to not include the .msix, but as it currently stands this appears to be the best way to have an accessible and efficient inner loop for the installer.